<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIENTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_registrar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textNombre">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textApellido1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textApellido2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textDNI">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEmail">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_mostrar_usuarios">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar usuarios</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaUsuarios">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textIdDelete">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Introduce la ID del cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_eliminar_cliente">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
